rpc-rs/src/chunkify.rs

```diff
@@ -19,34 +19,30 @@
 
 use std::{
     collections::HashMap,
-    io::Read,
     sync::{Arc, RwLock},
 };
 
-use nats::{
-    jetstream::JetStream,
+use async_nats::jetstream::{
+    self,
     object_store::{Config, ObjectStore},
-    JetStreamOptions,
+    Context,
 };
 use once_cell::sync::OnceCell;
 use tracing::{debug, error, instrument};
 
-use crate::{
-    error::{RpcError, RpcResult},
-    provider_main::get_host_bridge,
-};
+use crate::error::{RpcError, RpcResult};
 
 /// Maximum size of a message payload before it will be chunked
 /// Nats currently uses 128kb chunk size so this should be at least 128KB
-const CHUNK_THRESHOLD_BYTES: usize = 1024 * 700; // 700KB
+const CHUNK_THRESHOLD_BYTES: usize = 1024 * 900; // 900KB
```

Comment:
We should make sure to document this change in the changelog for history sake

Reply:
ok